examples/2-subdivision-number-in-header.tex

```diff
@@ -0,0 +1,282 @@
+\versenumber{60} Duis fringilla laoreet nisi, at suscipit ex porttitor sed.
+\versenumber{61} Mauris sit amet purus efficitur, euismod sem vitae, mattis neque.
+\versenumber{62} Mauris pretium, turpis a suscipit pulvinar, est nunc lacinia lacus, at pulvinar nunc lacus in erat.
+\versenumber{63} Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+\versenumber{64} Morbi nec augue elit.
+\versenumber{65} Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Quisque cursus eros a erat placerat, et tempus neque volutpat.
+\versenumber{66} Morbi ultrices at magna et cursus.
+\versenumber{67} Sed vel purus luctus, euismod erat in, fermentum orci.
+\versenumber{68} Vivamus ac rutrum sapien, id ultricies augue.
+\versenumber{69} Fusce sed rutrum lorem, ut auctor urna.
+\versenumber{70} Proin rutrum dignissim turpis, placerat pellentesque nulla elementum in.
+\versenumber{71} posuere cubilia Curae; Quisque cursus eros a erat placerat, et tempus neque volutpat.
+\versenumber{72} Morbi ultrices at magna et cursus.
+\versenumber{73} Sed vel purus luctus, euismod erat in, fermentum orci.
+\versenumber{74} Vivamus ac rutrum sapien, id ultricies augue.
+\versenumber{75} Fusce sed rutrum lorem, ut auctor urna.
+\versenumber{76} Proin rutrum dignissim turpis, placerat pellentesque nulla elementum in.
+\versenumber{77} Id pretium urna tortor sed purus.
+\versenumber{78} Donec velit leo, varius ac dui eget, tincidunt euismod nibh.
+\versenumber{79} Nulla facilisis sit amet leo et vehicula.
+\versenumber{80} Morbi ultrices elementum nisi, ac semper neque faucibus non.
+\versenumber{81} Curabitur et justo gravida eros consequat elementum.
+\versenumber{82} Quisque eu consequat eros, vitae ullamcorper risus.
+\versenumber{83} Nullam pellentesque id lectus a aliquet.
+\versenumber{84} Cras rhoncus, arcu ut consequat ultricies, massa ante sollicitudin massa, vehicula ullamcorper quam libero vitae eros.
+\versenumber{85} In dapibus arcu sed sem vehicula, ut imperdiet odio hendrerit.
+\versenumber{86} Praesent nec diam turpis.
+\versenumber{87} Vestibulum erat ante, maximus id pulvinar vitae, sagittis sed odio.
+\pend
+\endnumbering
+\newpage
+
+\begin{english}
+\section{Automatic verse number}
+
+\begin{abstract}
+ In this example, the verse (sentence) number is not typeset directly, but store in a \LaTeX\ counter (which name is \verb+verse+).
+ The \verb+\autoversenumber+ macro:
+ \begin{itemize}
+   \item increases the counter;
+   \item typesets it with a bold fontface, following by a dot;
+   \item use \verb+\doinsidelinehook+ to call correctly \verb+\markboth+.
+ \end{itemize}
+
+ The main problem is that, because of \emph{reledmac} mechanism,  the lines are typeset after the \verb+\pend+, 
+ at a time when the \verb+verse+ counter is already set to its maximal value.
+ Consequently, with hack, the \verb+\markboth+ macro won't have set the good value to the marker.
```

Comment:
with the hack, the \verb+\markbox+ macro won't have set the marker to the correct value.